.rubocop_todo.yml

```diff
@@ -348,9 +348,7 @@ Security/MarshalLoad:
 
 # Offense count: 2
 Style/AutoResourceCleanup:
-  Exclude:
-    - 'lib/rvg/to_c.rb'
-    - 'spec/image_list_2_spec.rb'
+  Enabled: false
```

Comment:
Would it be possible to fix this violation instead?

Reply:
@dlemstra I've got a followup chain in progress where I will re-enable a lot of rubocop rules after all of the spec files are split up. I didn't want to get distracted by it here, though, since as soon as I start refactoring, I might not stop. And then we'd end up in a half-way state with these tests forever.

Reply:
Thanks for the explanation. Maybe you could only exclude the files that have this issue instead of completely disabling the rule?

Reply:
Sure, I updated it.